### PR TITLE
fix: indent .codecov.yml correctly

### PR DESCRIPTION
**Pull request by AlekSi**
Indentation was wrong.

### CODE REVIEW

File: internal/output/codecov/codecov.go [hunk 1]
```diff
@@ -6,6 +6,7 @@
 package codecov
 
 import (
+	_ "embed" // for using go:embed
```
Comment (AlekSi) [inline internal/output/codecov/codecov.go:9]: Can't make it pass `make lint`

**Review by smira — COMMENTED** [review]
can you run it against itself? 

with `make kres`, `_out/kres gen` ? if there was a problem with indents, it should show up in Kres repo as well

**Comment by AlekSi** [comment]
```
# _out/kres gen
gen started
GET https://api.github.com/repos/AlekSi/talos-kres/branches/master/protection: 401 Bad credentials []
```

But the problem with indentation is there:

https://github.com/talos-systems/kres/blob/29565d0e086d0831de99c79ef48958f766bdab43/.codecov.yml#L5-L6

that should be

```yaml
codecov: 
    require_ci_to_pass: false 
```

The same for `status`.

**Comment by smira** [comment]
> ```
> GET https://api.github.com/repos/AlekSi/talos-kres/branches/master/protection: 401 Bad credentials []
> ```

why does it go into your fork? I think we have more stuff to fix it, but as a workaround, can you please try unsetting GITHUB_TOKEN? it will skip updating GitHub, but will generate new `.codecov.yml`

**Review by smira — APPROVED** [review]

**Pull request merged**